tests/cmdline/commands/test_profile.py

```diff
@@ -162,6 +162,35 @@ def test_delete_force(run_cli_command, mock_profiles, pg_test_cluster):
     assert 'When the `-f/--force` flag is used either `--delete-data` or `--keep-data`' in result.output
 
 
+@pytest.mark.parametrize('entry_point', ('core.sqlite_dos', 'core.sqlite_zip'))
+def test_setup_with_validating_sqlite_version(
+    config_psql_dos, run_cli_command, isolated_config, tmp_path, entry_point, monkeypatch
+):
+    """Test the ``verdi profile setup`` command.
+    Same as `test_setup`, here we check the functionality to check sqlite versions, before setting up profiles.
+    Note that this test should be run before the `test_delete_storage` test.
```

Comment:
Why?

Reply:
It was just to know if `profile setup` is failing, then I won't check why `test_delete_storage` is failing.
(Only because `test_delete_storage`  creates profiles, also with the `sqlite` backend.)
I've the habit of putting them in such order, maybe it doesn't matter that much...

I took the comment away, as this is not a most just easier to debug this way..